src/wallet/rpcwallet.cpp

```diff
@@ -2364,6 +2365,64 @@ UniValue settxfee(const JSONRPCRequest& request)
     return true;
 }
 
+UniValue setprivatesendrounds(const JSONRPCRequest& request)
+{
+    if (!EnsureWalletIsAvailable(request.fHelp))
+        return NullUniValue;
+
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 1)
+        throw std::runtime_error(
+            "setprivatesendrounds rounds\n"
+            "\nSet the number of rounds for PrivateSend mixing.\n"
+            "\nArguments:\n"
+            "1. rounds         (numeric, required) The default number of rounds is " + std::to_string(DEFAULT_PRIVATESEND_ROUNDS) + 
+            " Cannot be more than " + std::to_string(MAX_PRIVATESEND_ROUNDS) + " nor less than " + std::to_string(MIN_PRIVATESEND_ROUNDS) +
+            "\nResult:\n"
+            "true|false        (boolean) Returns true if successful\n"
+            "\nExamples:\n"
+            + HelpExampleCli("setprivatesendrounds", "4")
+            + HelpExampleRpc("setprivatesendrounds", "16")
+        );
+
+    LOCK2(cs_main, pwalletMain->cs_wallet);
```

Comment:
Not needed. We should probably have some PS client specific lock but this is out of the scope of this PR.